src/test/PoolSwapTest.sol

```diff
@@ -58,36 +57,48 @@ contract PoolSwapTest is Test, PoolTestBase {
         (,, uint256 reserveBefore1, int256 deltaBefore1) =
             _fetchBalances(data.key.currency1, data.sender, address(this));
 
-        assertEq(deltaBefore0, 0);
-        assertEq(deltaBefore1, 0);
+        require(deltaBefore0 == 0, "deltaBefore0 is not equal to 0");
+        require(deltaBefore1 == 0, "deltaBefore1 is not equal to 0");
 
         BalanceDelta delta = manager.swap(data.key, data.params, data.hookData);
 
         (,, uint256 reserveAfter0, int256 deltaAfter0) = _fetchBalances(data.key.currency0, data.sender, address(this));
         (,, uint256 reserveAfter1, int256 deltaAfter1) = _fetchBalances(data.key.currency1, data.sender, address(this));
 
-        assertEq(reserveBefore0, reserveAfter0);
-        assertEq(reserveBefore1, reserveAfter1);
+        require(reserveBefore0 == reserveAfter0, "reserveBefore0 is not equal to reserveAfter0");
+        require(reserveBefore1 == reserveAfter1, "reserveBefore1 is not equal to reserveAfter1");
 
         if (data.params.zeroForOne) {
             if (data.params.amountSpecified < 0) {
                 // exact input, 0 for 1
-                assertEq(deltaAfter0, data.params.amountSpecified);
-                assertGt(deltaAfter1, 0);
+                require(
+                    deltaAfter0 == data.params.amountSpecified,
+                    "deltaAfter0 is not equal to data.params.amountSpecified"
+                );
+                require(deltaAfter1 > 0, "deltaAfter1 is not greater than 0");
             } else {
                 // exact output, 0 for 1
-                assertLt(deltaAfter0, 0);
-                assertEq(deltaAfter1, data.params.amountSpecified);
+                require(deltaAfter0 < 0, "deltaAfter0 is not lesser than zero");
```

Comment:
another "lesser" here - please can you replace all the "lesser than" with "less than" ?